pfb-analysis/connectivity/reachable_roads_high_stress_calc.sql

```diff
@@ -1,7 +1,8 @@
 ----------------------------------------
 -- INPUTS
 -- location: neighborhood
--- maximum network distsance: 10560 ft
+-- :nb_boundary_buffer psql var must be set before running this script,
+--      e.g. psql -v nb_boundary_buffer=11000 -f connected_census_blocks.sql
```

Comment:
Worth mentioning here, I think, that the units are those of the the coordinate system of the `neighborhood_ways` tables.